Library/Formula/xsd.rb

```diff
@@ -0,0 +1,78 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/xsd-4.0.0+dep.tar.bz2"
+  sha256 "eca52a9c8f52cdbe2ae4e364e4a909503493a0d51ea388fc6c9734565a859817"
+  version "4.0.0"
+
+  depends_on "gcc" => :build
+  depends_on "findutils" => :build
+  depends_on "pkg-config" => :build
+  depends_on "xerces-c"
+
+  patch :DATA # Version 4.0.0 has some problems with Clang (see http://www.codesynthesis.com/pipermail/xsd-users/2015-February/004522.html )
+
+  def install
+    ENV.append "LDFLAGS", `pkg-config --libs --static xerces-c`.chomp
```

Comment:
Indeed. If I don't set LDFLAGS, make fails complaining that it doesn't know how to make target "-lxerces-c".